### ISSUE TITLE
USD build can't find PyOpenGL

### ISSUE THREAD

**Issue by emilche** [PixarAnimationStudios/USD#822]
### Description of Issue
I am trying to build USD but it complains that it can not find PyOpenGL. I have done the pip install PyOpenGL and it is in my site-package folder. 

-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   program_options
--   python
-- Found PySide: with C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/Common7/IDE/python, will use C:/Python27/Scripts/pyside-uic.exe for pyside-uic binary
CMake Error at C:/Program Files/CMake/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find PyOpenGL (missing: PYOPENGL_AVAILABLE)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindPyOpenGL.cmake:43 (find_package_handle_standard_args)
  cmake/defaults/Packages.cmake:138 (find_package)
  CMakeLists.txt:34 (include)

### System Information (OS, Hardware)
Windows 10, RTX 2080 TI.
### Build Flags
python C:\USD\build_scripts\build_usd.py --maya "D:\Program Files\USD"
[log.txt](https://github.com/PixarAnimationStudios/USD/files/3084804/log.txt)

**Comment by jilliene** [comment]
Filed as internal issue #USD-5206

**Comment by sunyab** [comment]
Hi @emilche, it looks like you're using a version of Python installed with Visual Studio to run the build script:

```
-- Found PythonInterp: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/Common7/IDE/python (Required is at least version "2.7") 
```

 My guess is that that version of Python is not looking at the site-packages directory where pip installed PyOpenGL, which is probably in C:\Python27. You could try using the Python interpreter from that directory to run the build script instead. Also, make sure that your Python installation is the 64-bit version.

**Comment by PixelPartner** [comment]
The 19.05 release [change log build section mentions](https://github.com/PixarAnimationStudios/USD/blob/master/CHANGELOG.md#build) a way to build with the Maya-build-in python (that lacks PyOpenGL)

**Comment by c64kernal** [comment]
@emilche hope you were able to resolve this.  Closing for now, feel to reopen if you're still running into trouble.

**Comment by emilche** [comment]
Sorry for not responding busy time at work... Today I was able to try any solutions and the problem was that it was using visual studios python as @sunyab said. I fixed it by editting the CmakeCache textfile and scrolled down to where it said C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/Common7/IDE/python and changed that to C:/Python27/python

This made it work. Thank you for all the help!
